docs/jwst/cfg_deprecation.rst

```diff
@@ -0,0 +1,85 @@
+.. _cfg_usage_deprecation_notice:
+
+CFG Usage Deprecation Notice
+============================
+
+As of March 18, a significant change to how JWST pipelines operate was completed
```

Comment:
For the sake of future-proofing this, you should add the year to the date.

Reply:
✅